snapcraft.yaml

```diff
@@ -41,6 +41,14 @@ description: |
                target: ubuntu-app-platform
          and then make your apps use it:
            plugs: [platform]
+       - access to Mir:
+           plugs:
+             mir:
+             mir-libs:
+               content: mir-libs
```

Comment:
As content == plug-name, it can be omitted (yeah, I tested and it even works, despite the content name non detection :p).

To make life easier for developers (as we did with other content interface plugs do you mind removing the `content` line?